map_gen/maps/crash_site/outpost_builder.lua

```diff
@@ -1447,11 +1481,15 @@ Public.magic_item_crafting_callback =
         local recipe = callback_data.recipe
         if recipe then
             entity.set_recipe(recipe)
+            if not callback_data.keep_active then -- to avoid trying to put an item into a fluid inventory
```

Comment:
You missed this when changing the table field name.
```suggestion
            if not callback_data.has_fluid_output then -- to avoid trying to put an item into a fluid inventory
```